products/salt.md

```diff
@@ -8,7 +8,6 @@ alternate_urls:
 -   /saltstack
 versionCommand: salt --version
 releasePolicyLink: https://docs.saltproject.io/salt/install-guide/en/latest/topics/salt-version-support-lifecycle.html
-releaseImage: https://docs.saltproject.io/salt/install-guide/en/latest/_images/salt-release-timeline.png
 changelogTemplate: https://docs.saltproject.io/en/__RELEASE_CYCLE__/topics/releases/__LATEST__.html
```

Comment:
```suggestion
changelogTemplate: https://docs.saltproject.io/en/latest/topics/releases/__LATEST__.html
```

Reply:
and removing all link tags for other cycles

Reply:
Nope, this would produce a dead link for 3006.9 : https://docs.saltproject.io/en/latest/topics/releases/2006.9.html.

Reply:
it is a missing file i will prepare a PR for them for 3006.9 until that time we can add link just for 3006.9

Reply:
If you want, but it also may be caused by their release process and in this case won't be fixed.

In the mean time wouldn't it be better to fix the links on endoflife.date ?

Reply:
sure :D lets merge this then when they fix their link we change our changelogtemplate

Reply:
for the record  https://github.com/saltstack/salt/issues/67139#issuecomment-2585880068

Reply:
@marcwrobel looks like they fixed problem on upstream : https://github.com/saltstack/salt/issues/67139#issuecomment-2744321751